pkg/bootkube/create.go

```diff
@@ -44,6 +46,9 @@ func CreateAssets(manifestDir string, timeout time.Duration) error {
 	if err := wait.Poll(5*time.Second, timeout, createFn); err != nil {
 		return fmt.Errorf("Failed to create assets: %v", err)
 	}
+
+	fmt.Println("Spawning self-hosted control plane...")
```

Comment:
I'd rather not add some kind of state around a log message. A common deployment is still going to just use `bootkube start` which means any effort towards layout of messages is going to be lost in the mix anyway. I'd prefer that the verbiage is changed to something along the lines of "this step is complete" or just removed.